src/savers/tvg/tvgTvgSaver.cpp

```diff
@@ -276,21 +273,29 @@ TvgBinCounter TvgSaver::serializeShape(const Shape* shape)
 {
     writeTag(TVG_TAG_CLASS_SHAPE);
     reserveCount();
+    TvgBinCounter cnt = 0;
 
     //fill rule
-    auto flag = (shape->fillRule() == FillRule::EvenOdd) ? TVG_FLAG_SHAPE_FILLRULE_EVENODD : TVG_FLAG_SHAPE_FILLRULE_WINDING;
-    auto cnt = writeTagProperty(TVG_TAG_SHAPE_FILLRULE, SIZE(TvgBinFlag), &flag);
+    if (shape->fillRule() == FillRule::EvenOdd) {
```

Comment:
Fixed condition.... it needs a comment just in case if the FillRule is expanded... So that we can easily notice this to fix.